db/migrate/20170606223403_drop_pages.rb

```diff
@@ -0,0 +1,5 @@
+class DropPages < ActiveRecord::Migration[5.0]
```

Comment:
@VivianChu Could we delete 20170322200503_create_pages.rb and 20170403173356_rename_to_slug.rb in folder db/migration instead? It looks like that we no longer need these two files.